main.tf

```diff
@@ -8,6 +8,11 @@ locals {
       "telemetry"  = 8006,
       "status"     = 8100
     }
+    portal = {
+      "portal_gui" = 8446,
+      "portal_api" = 8447,
```

Comment:
these are HTTPS ports, but the control plane above is using HTTP ports. Why the difference?

Reply:
Which ports are you looking at? 8444/8445 are HTTPS, 8005 and 8006 are used for websockets